app/assets/javascripts/initializePage.js

```diff
@@ -54,8 +54,6 @@ function initializePage() {
         initializeAllTagEditButtons();
       }
       initializeBroadcast();
-      initializeAllFollowButts();
```

Comment:
Bye bye butts. 🤣

Reply:
I feel quite sorry to remove something that has brought so much joy 😆